src/middlewares/AuthenticateUser.js

```diff
@@ -8,14 +8,11 @@ class AuthenticateUser {
   static async checkUserArticle(req, res, next) {
     try {
       const user = await User.findOne({ where: { email: req.user.email } });
-      !user && res.status(403).json({ message: 'User not found!' });
-
+      if (!user) { return res.status(403).json({ message: 'User not found!' }); }
```

Comment:
remove curly brackets because the inner statement is a single line ==> single block